appinventor/components/src/com/google/appinventor/components/runtime/File.java

```diff
@@ -319,6 +326,17 @@ public void GotText(String text) {
   }
 
   /**
+   * Event indicating that a request has finished.
+   *
+   * @param text write to the file
+   */
+  @SimpleEvent (description = "Event indicating that the contents of the file have been written.")
+  public void AfterFileSaved(String text) {
```

Comment:
text --> fileName

This should be fileName rather than filename, since the translation system already has translations for "fileName" in it.   To test this, switch the language to Chinese and check that the parameter name in the block has been translated (as it is in the File.Delete.